package.json

```diff
@@ -23,11 +23,10 @@
   },
   "scripts": {
     "build": "gulp",
-    "pretest": "npm run lint",
-    "test": "gulp test",
-    "html-report": "nyc report --temp-directory=./coverage --reporter=html",
-    "jest": "jest --coverage",
-    "coveralls": "nyc report --temp-directory=./coverage --reporter=text-lcov | coveralls",
```

Comment:
@arunoda coveralls needs to be piped the `lcov` format test output post test:

https://github.com/zeit/next.js/blob/master/.travis.yml#L14

so this logic is necessary.